Submit Assignment/Darshan-Kawade-76/css/style.css

```
@@ -0,0 +1,474 @@
+    background-color: white;
+    width: 100%;
+    height: 100%;
+}
+
+/* Front Side */
+.front {
+    left: -20pt; /* 0px */
+    margin-top: -0.69pt; /* -0.92px */
+    width: 59.002mm;
+    height: 83.872mm;
+    margin: 6.085mm;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)